### ISSUE TITLE
About the performance

### ISSUE THREAD

**Issue by TonyPod** [da2so/Dreaming-to-Distill-Data-free-Knowledge-Transfer-via-DeepInversion-Tensorflow#1]
Hi there, thank you for the great Tensorflow 2 implementations! It helped me a lot since recently I am also struggling to implement a Tensorflow 2 version of this paper.
I find that there is a wrong usage of loss in your code: in [Line 96 of DeepInversion](https://github.com/da2so/Dreaming-to-Distill-Data-free-Knowledge-Transfer-via-DeepInversion-Tensorflow/blob/62fa7a578e3c3186d7a5ecdc310acfcadf433ded/DeepInversion.py#L96), the positions of two parameters should be swapped according to this [Tensorflow document](https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy).
However, after fixing it the generated images are still not as good as the PyTorch version released by the authors. I don't know if you have some ideas to further improve the quality of the images. Thank you!
